package.json

```diff
@@ -159,12 +193,20 @@
           "inventory-storage.location-units.libraries.collection.get",
           "inventory-storage.service-points.collection.get",
           "inventory-storage.service-points.item.get",
+          "circulation-storage.staff-slips.collection.get"
+        ],
+        "visible": false
+      },
+      {
+        "permissionName": "ui-tenant-settings.settings.servicepoints",
+        "displayName": "Settings (tenant):  Can view service points",
```

Comment:
This displayName should probably be made "Settings (tenant):  Can create, edit and remove service points"??
And also update "visible" to "true".

Reply:
Please also add new permission display names to `en.json` so they can be translated

Reply:
@Terala-Priyanka thanks, good catch. 
@BogdanDenis this one should not be added because it will not shown to the users, but I've updated another one newly added. Thanks for catching.

Reply:
@vashjs
I suggest the translations still be defined. ATM it might not be visible. But it can be made visible in future.